src/Item_Ticket.php

```diff
@@ -198,6 +198,7 @@ public static function itemAddForm(Ticket $ticket, $options = [])
         ];
 
         $opt = [];
+        unset($options['entities_id']);
```

Comment:
Why this ?

Reply:
Wrong piece of code related to: https://github.com/glpi-project/glpi/pull/15450 
This bug was solved in another way. I've just suppress the line.